Controller/CRUDController.php

```diff
@@ -221,7 +221,8 @@ public function deleteAction($id)
     /**
      * Edit action.
      *
-     * @param int|string|null $id Optional ID to use instead of globally provided default.
+     * @param int|string|null $id Optional ID representing the main admin subject.
```

Comment:
> to use instead of globally provided default

@rande @Soullivaneuh : I guess that's why this needs to be so complicated?
@Soullivaneuh : why are you removing this comment?
@rande : I fail to see in which use case `$id` needs to be used. Do you know why this is needed ?